### PR TITLE
Document auto-reboot and others for secondaries

### PR DESCRIPTION
**Pull request by lbonn**

### CODE REVIEW

File: docs/ota-client-guide/modules/ROOT/pages/posix-secondaries-bitbaking.adoc [hunk 1]
```diff
@@ -80,6 +80,8 @@ bitbake secondary-image
 |An update type to configure for Secondary, supported types: ostree, file
 |===
 
+Secondary images will also take advantage of some common configuration snippets described in xref:recommended-clientconfig.adoc[Recommended client configurations], such as auto-reboot, persistent journald and debug logs.
```
Comment (ammaboateng) [inline docs/ota-client-guide/modules/ROOT/pages/posix-secondaries-bitbaking.adoc]:
```suggestion
Secondary images will also take advantage of some common configuration snippets described in xref:recommended-clientconfig.adoc[recommended client configurations], such as auto-reboot, persistent `journald` and debug logs.
```

Comment (lbonn) [reply docs/ota-client-guide/modules/ROOT/pages/posix-secondaries-bitbaking.adoc]: thanks, applied

**Comment by lbonn** [comment]
Let's wait until we merge https://github.com/advancedtelematic/meta-updater/pull/696

**Comment by tkfu** [comment]
> Let's wait until we merge [advancedtelematic/meta-updater#696](https://github.com/advancedtelematic/meta-updater/pull/696)

Sure, if you prefer. Note that, because this is going to be merged to master (and not 2020.3-docs), it won't actually be published on the docsite until the 2020.4 release.

**Pull request merged**